app/assets/stylesheets/atmosphere/sections/tables.scss

```diff
@@ -0,0 +1,13 @@
+th.table-cell-uuid {
```

Comment:
Are you sure `table-cell` prefix is required in this kind of css elements? Maybe `th.uuid`, `th.monospace` is enough?